site/content/docs/main/_index.md

```diff
@@ -198,58 +201,36 @@ through the [issues][issue] and jump in.
 
 ### Before you start
 
-* Please familiarize yourself with the [Code of Conduct][coc] before contributing.
-* See [CONTRIBUTING.md][contrib] for instructions on the developer certificate of origin that we require.
-* There is a [Slack channel][slack] if you want to interact with other members of the community
+- Please familiarize yourself with the [Code of Conduct][coc] before contributing.
+- See [CONTRIBUTING.md][contrib] for instructions on the developer certificate of origin that we require.
+- There is a [Slack channel][slack] if you want to interact with other members of the community
 
 ## Changelog
 
 See [the list of releases][releases] to find out about feature changes.
 
 [decoupling-sonobuoy-k8s]: https://sonobuoy.io/decoupling-sonobuoy-and-kubernetes
-
 [airgap]: airgap
-
 [brew]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-with-homebrew-on-macos
-
 [cncf]: https://github.com/cncf/k8s-conformance#certified-kubernetes
-
 [coc]: https://github.com/vmware-tanzu/sonobuoy/blob/main/CODE_OF_CONDUCT.md
-
 [contrib]: https://github.com/vmware-tanzu/sonobuoy/blob/main/CONTRIBUTING.md
-
 [docker]: https://docs.docker.com/get-docker/
-
 [docs]: https://sonobuoy.io/docs/main
-
-[e2ePlugin]: e2eplugin
-
-[customPlugins]: plugins
-
+[e2eplugin]: e2eplugin
+[customplugins]: plugins
 [gen]: gen
-
 [issue]: https://github.com/vmware-tanzu/sonobuoy/issues
-
 [issue1388]: issue1388
-
 [k8s]: https://github.com/kubernetes/kubernetes
-
 [kind]: https://github.com/kubernetes-sigs/kind
-
 [linux]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1
-
 [oview]: https://youtu.be/8QK-Hg2yUd4
-
 [plugins]: plugins
-
 [quickstart]: https://aws.amazon.com/quickstart/architecture/vmware-kubernetes/
-
 [releases]: https://github.com/vmware-tanzu/sonobuoy/releases
-
 [results]: results
-
 [slack]: https://kubernetes.slack.com/messages/sonobuoy
-
-[snapshot]:snapshot
-
```

Comment:
I'd prefer you remove all these non-impactful changes: 71 changes to this file but I think only 3 of them are necessary. The others are changing `*` to `-`, changing the casing of the markdown link name which no one sees, and whitespace which doesnt change anything either.

Not that those things are wrong to change, but typically you want to try and put those sort of 'no-impact' changes in their own PR where its easy to check that things are correct. Otherwise its hard to accidentally slip something else into this PR you dont mean to.

Reply:
@johnSchnake These changes were automatically added when I ran `./scripts/sync_readme.sh` in the repo.

Reply:
@johnSchnake These 'no-impact' changes were automatically added when I ran ./scripts/sync_readme.sh in the repo. So, should I remove them or not ?